Automattic/sensei
# Add tracking for link to "Edit Lesson" from course

Closes #2659 

Tracks clicking the link to edit a lesson from the course page. This link is in the "Course Lessons" metabox.

The implementation builds off of https://github.com/Automattic/sensei/pull/2697. This PR will be a draft until #2697 is merged.

## Testing Instructions

- Open the Edit page for a course, and click the "Edit Lesson" link in the Course Lessons metabox beside a lesson. The event should be logged.
- Try this in both the Classic and Block editors.
- Try changing the course's status and clicking the link again. Ensure that the `course_status` attribute is logged correctly.
- Ensure the click logging for the Add Lesson link still works. See https://github.com/Automattic/sensei/pull/2697
